rfcs/react-components/components/combobox-spacebar.md

```diff
@@ -0,0 +1,115 @@
+
+One of the risks is that the developers and designers may not be in a position to make a correct decision on the value of the prop. This is because options are usually either translations or user data, which may contain spaces that the authoring team was not aware of. We generally can't expect authors to understand all possible translations across supported languages or the specific validations of the user data at UI authoring time. While the occaisonal space character does not necessarily mean that space should not select, translations and user data do introduce the possibility of frequent spaces within multiple options.
+
+## Problem statement
+
+We cannot fully know whether the ideal spacebar behavior is to select or insert a character, since we don't know the full context of usage. Our choices mostly revolve around balancing providing more flexibility vs. more consistency, and choosing the best default if we do provide flexibility.
+
+## Options
+
+### 1. `allowSpaceCharacter` prop, default to `true`
+
+This is the option that allows authors to choose the best behavior for their use case, defaulting to allowing space to insert a character. This borrows the behavior from the freeform PR, where space will still select if the user is actively interacting with the dropdown listbox (i.e. it is open, and the last interaction was to navigate through options).
+
+We would likely include author guidance that all Comboboxes within their app use the same behavior for `allowSpaceCharacter`.
+
+#### Pros
+
+- Allows authors to choose the best behavior per use case
+- Defaulting to `true` means the default probably(?) matches the majority of Combobox usage
+- Defaulting to `true` means that often-missed use cases like spaces in translated strings or user-generated options won't cause issues
+
+#### Cons
+
+- Defaulting to `true` means the simplest/most basic use case is not the default
+- Defaulting to `true` means the Combobox space behavior is different from Dropdown by default, which primarily affects keyboard and screen reader users.
+- Providing a prop to alter the space key behavior potentially means inconsistent behavior within an app, or across related apps
+- This behavior is not aligned with the results of the user study
+
+### 2. `allowSpaceCharacter` prop, default to `false`
+
+This is the same as (1), but with a different default. The Pros/Cons list is essentially the same, but reversed.
+
+### 3. Always allow space to insert a character
+
+We would keep the behavior where if the user is actively interacting with the options in the dropdown listbox, space will select. Otherwise, space will insert a character in the input.
+
+#### Pros
+
+- Authors don't need to try to figure out what the ideal space key behavior is on a case-by-case basis / less authoring overhead for an already complex component
+- Consistent behavior across all apps using Fluent
+
+#### Cons
+
+- There is no way to opt into space-to-select behavior
```

Comment:
Agreed, since there is already the precendence of not using space to select in v8 and v0, that should mean that anyone that is enabling that in those libraries had to have implemented that themselves using event handling. That code is already there and most likely won't change too much when adopting the V9 combobox